ui/src/components/navigation/Sidebar.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { FC, ReactElement, useState, useEffect } from 'react';
+import {
+  Box,
+  Drawer,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+  useMediaQuery,
+} from '@mui/material';
+import HomeIcon from '@mui/icons-material/Home';
+import MusicNoteIcon from '@mui/icons-material/MusicNote';
+import QueueMusicIcon from '@mui/icons-material/QueueMusic';
+import SupervisedUserCircleIcon from '@mui/icons-material/SupervisedUserCircle';
+import TextSnippetIcon from '@mui/icons-material/TextSnippet';
+
+interface SidebarProps {
+  isOpen: boolean;
+}
+
+const drawerWidth = 240;
+
+const SideBar: FC<SidebarProps> = ({ isOpen }): ReactElement => {
+  const [selectedItem, setSelectedItem] = useState<string | null>(null);
+  const isDesktop = useMediaQuery('(min-width: 769px)');
+  const menuItems = [
+    { icon: <HomeIcon />, text: 'Home' },
+    { icon: <MusicNoteIcon />, text: 'Songs' },
+    { icon: <QueueMusicIcon />, text: 'My Setlists' },
+    { icon: <SupervisedUserCircleIcon />, text: 'Groups' },
+    { icon: <TextSnippetIcon />, text: 'Resources' },
+  ];
+
+  // Reset when switching views
```

Comment:
It's reset when I change from mobile to desktop, but idt its reset when I change from desktop to mobile?